util/ubuntu-setup

```diff
@@ -73,6 +76,34 @@ if [ $UID -ne 0 ]; then
 	exec sudo $0 $@
 fi
 
+# ask for permission to modify apt sources if '-y' flag abesnt
+if [ "ubuntu-18.04" = "$SYSTEM" -a "$YES" != true ]; then
+	echo "Need to add xenial universe to package lists for arm compiler support."
+	read -r -p "Is this okay? [y/N] " response
+	case "$response" in 
+	[yY][eE][sS]|[yY]) 
+        	;;
+    	*)
+		echo "cannot continue on Bionic without permission to add to lists"
+		echo "-- exit"
+		exit 1
+        	;;
+	esac
+fi
+
+if [ "ubuntu-18.04" = "$SYSTEM" ]; then
+	SOURCE_LIST=/etc/apt/sources.list.d/gcc-arm-none-eabi_xenial.list
+	if [ ! -f $SOURCE_LIST ]; then
+		echo "-- adding xenial universe to support arm compiler"
+		touch $SOURCE_LIST
+		echo "# robocup-firmware added xenial universe for gcc-arm-none-eabi 5.4.1" > $SOURCE_LIST
```

Comment:
This can be much simpler with a [heredoc](http://www.tldp.org/LDP/abs/html/here-docs.html)

Also, you probably meant to do `>>` a line below, but it won't matter if you switch to heredocs :P